src/runtime/hexagon/hexagon_device_api.h

```diff
@@ -190,16 +194,15 @@ class HexagonDeviceAPI final : public DeviceAPI {
            (DLDeviceType(dev.device_type) == kDLCPU);
   }
 
-  //! \brief Manages underlying HexagonBuffer allocations
+  //! \brief Manages RPC HexagonBuffer allocations
+  // rpc_hexbuffs is used only in Alloc/FreeRpcDataSpace
+  std::unique_ptr<HexagonBufferManager> rpc_hexbuffs;
+
+  //! \brief Manages runtime HexagonBuffer allocations
```

Comment:
It looks like the primary purpose of the separation between `runtime_hexbuffs` and `rpc_hexbuffs` is to allow scoped allocations that can be uniformly cleaned up at the end of a scope.  Can we add a comment describing the purpose, along with why it is only being implemented for Hexagon?

Reply:
Yes, I'll add another comment.